README.md

```diff
@@ -19,9 +19,35 @@ To create a new project with the tool:
 apostrophe create <project name>
 ```
 
-If you run the `create` command with the `--install` flag, the command will also `npm install` the dependencies of the project and `apostrophe:reset` the database, resulting in a fully bootstrapped project.
+This will create a local copy of our standard [Apostrophe Boilerplate](https://github.com/punkave/apostrophe-boilerplate).
 
-#### Run command-line tasks
+If you run the `create` command with the `--install` flag, the command will also `npm install` the dependencies for the project and `apostrophe:reset` the database, resulting in a fully bootstrapped project.
+
+#### Create a widget
+To bootstrap the necessary files and basic configuration for a new Apostrophe widget, run the following command from within your Apostrophe project's root directory:
+```bash
+apostrophe createWidget <widgetName>
+```
+
+#### Create a piece
+To bootstrap the necessary files and basic configuration for a new Apostrophe piece type, run the following command from within your Apostrophe project's root directory:
+```bash
+apostrophe createPiece <pieceName>
+```
+
+If you run the `createPiece` command with the `--pages` flag, the command will also set up a corresponding pieces-pages module with your new piece type. Similarly, you can run the `createPiece` command with the `--widgets` flag, which will also set up a corresponding pieces-widgets module along with your new piece type. These flags can be used together or separately.
+
+```bash
+apostrophe createPiece <pieceName> --pages --widgets
+```
+
+#### Create an empty Apostrophe module
+To bootstrap the necessary files and basic configuration for a super basic Apostrophe module, run the following command from within your Apostrophe project's root directory:
```

Comment:
"A brand-new Apostrophe module that doesn't extend one of the usual suspects like pieces or widgets" might make more sense to people than "super basic".